question-service/requests.rest

```diff
@@ -1,6 +1,3 @@
-### CHECK SERVER ALIVE
```

Comment:
Might be good to keep and develop a test for this endpoint as a basic way to check for server status? What do you think?